Update workflows and license

JamyGolden opened the pull request:
- I removed people from the license who weren't you (since you created it in Feb) and Tinted Theming (the new owners) since they've never had any ownership of the repo. Let me know if I've misunderstood something here though.
- Add Auto assignment for issues action - This will simply assign the maintainers of the repo (just you) to the github issue (github doesn't automatically do this)
- Use the shared tinted-theming github action to update themes
- Add CODEOWNERS file

auxym commented:
All good for me, but should we keep Scott C. Lowe's copyright in the license? I based the mustache template on his color schemes from here https://github.com/scottclowe/matlab-schemer/tree/master/schemes

JamyGolden commented:
Ah ok, I've adjusted the license and readme. Since that project is under `BSD-2-Clause`, we're required to add that to the LICENSE file. I've also added credit in the readme.
